### PR TITLE
Ability to push ARM/AMD64 Operator UI images to GHCR

### PR DESCRIPTION
**Pull request by urashidmalik**
Updated githubs action workflow to build and push **Operator UI** container to GHCR
OperatorUIPullReq1

### CODE REVIEW

File: ui/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,5 @@
-FROM registry.opensource.zalan.do/library/alpine-3.15:latest
+ARG BASE_IMAGE=registry.opensource.zalan.do/library/alpine-3.15:latest
+FROM ${BASE_IMAGE}
```
Comment (FxKu) [inline ui/Dockerfile:2]: Why is this change needed?

**Comment by FxKu** [comment]
:+1:

**Comment by hughcapet** [comment]
👍

**Comment by FxKu** [comment]
Thanks @urashidmalik for your contribution. I will change a few little things in a follow PR.
